src/main/groovy/com/bmuschko/gradle/cargo/tasks/local/LocalCargoContainerTask.groovy

```diff
@@ -188,13 +189,19 @@ class LocalCargoContainerTask extends AbstractCargoContainerTask {
                 setContainerSpecificProperties()
 
                 getDeployables().each { deployable ->
+                    def deployableType = getDeployableType(deployable)
+
+                    if (DeployableType.EXPLODED == deployableType) {
+                        deployableType = DeployableType.WAR
```

Comment:
Is it required to use the type `war` for exploded deployables? It would make more sense to me if we wouldn't have to set the `type` attribute at all.